scripts/contracts/footer.js

```diff
@@ -0,0 +1,29 @@
+/*
+  `updateFooter` is specific to the Contracts table in `contracts/view.html`.
+  
+  It integrates with `smart-table.js` to be called when data is filtered.
+*/
+const updateFooter = filters => {
+  const productFilters = filters.filter(f => f.field === 'product' && f.type === 'like');
+  
+  const footnotes = $('footer').children('ol').children('li');

```

Comment:
I think my only requested change would be this line - the `$('footer')` selector seems too broad (we have 2 `<footer>` elements on the page already).

Can we use the footnotes list selector instead? `ol.footnote-listing`. That way we're sure to get the list of footnote items.

Reply:
Good suggestion 😄 - I changed it to `$('ol.footnote-listing').children('li')`

Reply:
https://github.com/cal-itp/mobility-marketplace/pull/325/files#diff-4b3572daf35019e63906407ff17148ff60f18b7cb4b0825555c1131e3ed40b5fR9